dwave/plugins/sklearn/transformers.py

```diff
@@ -0,0 +1,257 @@
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import SelectorMixin
+from dwave.system import LeapHybridCQMSampler
+import dimod
+import numpy as np
+import pandas as pd
+import tempfile
+
+import logging
+
+class HybridFeatureSelection(BaseEstimator, SelectorMixin):
+    
+    def __init__(self, alpha : float = .5, time_limit : int = 10, n_default_feature : int = 10, method = "correlation") -> None:
+        """
+        Initialize a hybrid quantum-classical feature selection transformer. 
+
+        Args:
+            alpha (float, optional): hyper-parameter which weights correlation to outcome vs covariance. Must be between 0 and 1. Defaults to .5.
+            time_limit (int, optional): a time limit for the Leap Constrained Quadratic Model Solver to solve the feature selection problem. 
+                                        Must be at least the minimum time limit of the solver. Defaults to 10.
+            n_default_features (int, optional)
+            method (string, optional): The method used to calculate the model matrix. One of "correlation" or "mutual information". Defaults to correlation.
+        """
+        super().__init__()
+        self.alpha = alpha
+        self.time_limit = time_limit
+        self.n_default_feature = n_default_feature 
+        self.selected_columns = None
+        self.mask = None
+        assert (self.alpha <= 1) and (self.alpha >= 0), f"alpha {self.alpha} is not between 0 and 1"
```

Comment:
Don't use asserts for input checking.
```suggestion
        if not 0 <= alpha <= 1:
            raise ValueError(f"alpha {self.alpha} is not between 0 and 1")
```